### PR TITLE
Travisci flake8

### PR DESCRIPTION
**Pull request by hramezani**
## Problem

Explain the problem you are fixing (add the link to the related issue(s), if any).

## Solution

Explain the solution that has been implemented, and what has been changed.

## Commandments

- [ ] Write PEP8 compliant code.
- [ ] Cover it with tests.
- [ ] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [ ] Pay attention to backward compatibility, or if it breaks it, explain why.
- [ ] Update documentation (if relevant).

### CODE REVIEW

**Comment by hramezani** [comment]
@smithdc1 Please take a look if you have time. Thanks.

File: tox.ini
Comment (smithdc1) [inline tox.ini:27]: Can we skip the install for this environment? It saves a bit of time from installing the package (and Django and it's dependencies too)

https://tox.readthedocs.io/en/latest/config.html#conf-skip_install

File: tox.ini
Comment (smithdc1) [inline tox.ini:30]: If this is also making a few amends to the tox file could change this to just `pytest`?

I also wonder about adding `-W error` so any deprecation warnings get caught earlier?

**Review by smithdc1 — COMMENTED** [review]
LGTM. Just a nit pick and a potentially unrelated comment.

**Comment by jezdez** [comment]
@hramezani  I'm planning on working to port django-model-utils to GitHub Actions from Travis ASAP, are you okay with updating the PR later once that's merged or do you want this to land first?

File: tox.ini
Comment (hramezani) [reply tox.ini:27]: Good idea. `skip_install` added.

File: tox.ini
Comment (hramezani) [reply tox.ini]: I changed it to `pytest`.

> I also wonder about adding -W error so any deprecation warnings get caught earlier?

I am not sure about this TBH.

**Comment by hramezani** [comment]
@jezdez I think this PR is ready to merge if @smithdc1 approves it.

**Review by smithdc1 — APPROVED** [review]

**Comment by smithdc1** [comment]
Approved. :+1:

**Pull request merged**
